pandera/model.py

```diff
@@ -521,6 +521,12 @@ def __modify_schema__(cls, field_schema):
         """Update pydantic field schema."""
         field_schema.update(to_json_schema(cls.to_schema()))
 
+    @classmethod
+    @docstring_substitution(validate_doc=DataFrameSchema.empty.__doc__)
+    def empty(cls: Type[TSchemaModel]) -> DataFrame[TSchemaModel]:
```

Comment:
Not too sure `DataFrame` is the right return type here - I thought that the signature would be the same as `validate`, but pycharm kept complaining, and I couldn't run mypy to make sure.